superset-frontend/src/cccs-viz/plugins/plugin-chart-cccs-grid/src/CccsGrid.tsx

```diff
@@ -404,9 +402,7 @@ export default function CccsGrid({
   };
 
   const onRangeSelectionChanged = (params: any) => {
-    if (params.finished === false) {
-      return;
-    }
+    console.log("calculating")
```

Comment:
Do we still need that log statement?

Reply:
not sure when these changes are from but I will remove all the changes from the old grid files since it isnt being used anymore